src/config/index.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   },
   INFURA_API_KEY: process.env.INFURA_API_KEY || "bb46da3f80e040e8ab73c0a9ff365d18",
   INFURA_PROJECT_ID: "1f1ff2b3fca04f8d99f67d465c59e4ef",
+  ALCHEMY_API_KEY: process.env.ALCHEMY_API_KEY || "pV9JmoYcDFFBihXlJWt6FSDxzNAVWxdT",
   IS_DEVELOPMENT,
```

Comment:
i presume this is to a dashboard where the api key is generated. do we have login access to this dashboard?

Reply:
currently its i am the one that created this api key, i talked to ZL and he said that we can use this for the time being and generate one with the govtech email in future.

Reply:
maybe let me create 1 with dlt gmail. let u noe again

Reply:
created and added in netlify variable, can leave yours there as fallback